docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1115,63 +1115,165 @@ For the purposes of our simulation, we made a few assumptions:
+simplifying assumptions:
+
+1.  The above two factors each reduce the reported Hb effect size by a multiplicative factor independent of the other factor.
+
+2.  Each multiplicative factor is a continuous piecewise linear function of time.
+
+We will model the age effect with a continuous piecewise linear function
+``hb_age_fraction(age)``, where ``age`` is the current age of the simulant, and
+we will model the time-lag effect with a continuous piecewise linear function
+``hb_lag_fraction(time_since_fortified)``, where ``time_since_fortified`` is the
+time since the simulant started eating iron-fortified food:
 
 .. code-block:: Python
 
-	# at the draw level
-	MD = hb_md_distribution.rvs() # mean difference in hemoglobin concentration due to iron fortification
-		# note, hb_md_distribution defined above in the effect size section
+  def hb_age_fraction(age):
+  """
+  Multiplier on Hb effect size due to children eating less food at younger
+  ages. (`age` is current age in years).
+  """
+    return 0 if age<0.5 else (age-0.5)/1.5 if age<2 else 1
+
+  def hb_lag_fraction(time_since_fortified):
+  """
+  Multiplier on Hb effect size due to lag in response time to iron fortification.
+  The argument `time_since_fortified` is the time (in years) since a simulant
+  started eating fortified food (note that a negative value of `time_since_fortified`
+  indicates the child has not yet started eating fortified food).
+  """
+    return (0                         if time_since_fortified < 0   else
+            time_since_fortified/0.5  if time_since_fortified < 0.5 else
+            1)
+
+.. Note::
+
+  Since children don't start eating fortified food until 6 months of age, the
+  argument ``time_since_fortified`` for the time-lag function will be *either*
+  the time since the child's household got fortification coverage, *or* the time
+  since the child turned 6 months old, *whichever is smaller*.
+
+.. Note::
+
+  When calibrating hemoglobin levels for baseline coverage (see the
+  :ref:`baseline calibration <baseline_calibration_hb_bw_section>` section
+  below), we will use **only** the age effect to compute the treatment-deleted
+  hemoglobin levels of the population (the down-shift), whereas we will use
+  **both** the age effect *and* the time-lag effect when applying the treatment
+  to an individual simulant (the up-shift). This will cause our population's
+  overall mean hemoglobin level to be slightly lower than the value in GBD, but
+  not by much, and this is the simplest strategy that seems reasonable.
+
+.. Note::
+
+  When calibrating hemoglobin levels for baseline coverage (see the
+  :ref:`baseline calibration <baseline_calibration_hb_bw_section>` section
+  below), it is *possible* (but highly unlikely with the shift size we're dealing with) that a simulant's hemoglobin level can end up *negative* after the down-shift. **In the case of a negative hemoglobin level, we will arbitrarily reset the hemoglobin level to 10 g/L**.
```

Comment:
That is pretty uncool---if it is .1, you leave it, but if it is -.1, you make it 10?  Better to clip everything to 10, instead: `hb_level = np.clip(orig_hb_level, 10, np.inf)`.

Reply:
Ok great, thanks for the suggestion @aflaxman . The number 10 was pretty arbitrary - is there any is there any better methodology to picking a cutoff?  Really we just need it to be positive, so what about using a cutoff of 0 (or some very small positive number)?  It seems like that would be more consistent with the *non*-iron-responsive individuals, whose Hb level doesn't get adjusted at all.

Reply:
Google thinks 10 g/L is a very small positive number:
![image](https://user-images.githubusercontent.com/51236/83702268-13579700-a5c1-11ea-907e-526fd4598eca.png)

Reply:
@aflaxman Well yes, that's why I arbitrarily chose 10 - I figured once it's that low, the actual value doesn't really matter. But there are several issues here:

1. If 10 g/L is absurdly low, why not truncate at 50, or 75, or some other number?  I'm saying we have no good way of choosing a good cutoff (without doing extra research to make sure we get it right), so why not just go with the only cutoff we know for sure, which is 0?

2. If we're going to try to make the cutoff for the shifted distribution more realistic, then should we modify the unshifted GBD data to have more realistic values as well?  Namely, non-iron-responsive simulants will have Hb levels according to the GBD ensemble distribution (40% gamma, 60% mirror Gumbel), which I believe is supported on all real numbers, hence could theoretically be negative.  If we're truncating the shifted version of this at 10 g/L, should we also truncate the unshifted version at 10g/L?  Why not 50, or 75, or some other value?

3. Doing this truncation isn't necessarily about making our model more realistic - it's a hack to correct for our dumb method of applying the effect size and baseline calibration, to make sure our model doesn't give us blatantly wrong results.  For the LSFF model, I think the probability of seeing a negative Hb value will be so low that we will never actually run into this issue, but I remember James saying they *did* get some negative birthweights in the BEP model (which had a larger shift), and it should be the researchers' job to decide how to fix that rather than the SWEs'.  So I guess I'm trying to figure out what should be our general methodology.

Reply:
A principled way to approach it would be to look at the RR and see if it gives us an answer.